includes/admin/wc-meta-box-functions.php

```diff
@@ -181,6 +181,33 @@ function woocommerce_wp_checkbox( $field ) {
 	echo '</p>';
 }
 
+
+/**
+ * Return the html selected attribute if stringified $value is found in array of stringified $options
+ * or if stringified $value is the same as scalar stringified $options.
+ *
+ * @param string|int       $value   Value to find within options.
+ * @param string|int|array $options Options to go through when looking for value.
+ * @return string
+ */
+function wc_selected( $value, $options ) {
+	$value = (string) $value;
+	if ( is_array( $options ) ) {
+		$options = array_map( 'strval', $options );
+		$is_value_in_options = in_array( $value, $options, true );
+	} else {
+		$options = (string) $options;
+		$is_value_in_options = ( $value === $options );
+	}
+
+	if ( $is_value_in_options ) {
+		return " selected='selected'";
```

Comment:
No need for double quotes here, we can just use `' selected="selected"';`. WP returns in single quotes because of https://developer.wordpress.org/reference/functions/__checked_selected_helper/ but we don't need it here.

Reply:
I think it's find to just return `selected` - thats valid HTML5